Sources/DeploymentTargetAWSLambda/AWSIntegration.swift

```diff
@@ -182,11 +182,15 @@ class AWSIntegration { // swiftlint:disable:this type_body_length
             try deploymentStructure.writeJSON(to: launchInfoFileUrl)
             try fileManager.setPosixPermissions("rw-r--r--", forItemAt: launchInfoFileUrl)
             
+            
             do {
                 // create & add bootstrap file
+                // TODO: Not entirly sure if this passing of ENVs works
                 let bootstrapFileContents = """
                 #!/bin/bash
-                ./\(lambdaExecutableUrl.lastPathComponent) --mode=\(WellKnownCLIArguments.launchWebServiceInstanceWithCustomConfig) --fileurl=./\(launchInfoFileUrl.lastPathComponent)
+                \(WellKnownEnvironmentVariables.executionMode)=\(WellKnownEnvironmentVariableExecutionMode.launchWebServiceInstanceWithCustomConfig)
```

Comment:
I don't see why this shouldn't work (but then i haven't tested it), but i think it might be a better idea to use lambda's environment handling to set these. (That's what we're already doing with the node id)

You can leave the code as is, and I'll make that change in my branch once this is merged

Reply:
👍